.drone.star

```diff
@@ -44,6 +45,7 @@ def main(ctx):
       'onlyoffice_sha': 'b4bef78d11938d3892c4c5993cab110c149e0aadef0e60639513d9c1c2568cb1',
       'php': '7.3',
       'base': 'v19.10',
+      'behat_version': 'v10.3.3',
```

Comment:
Note: there is a `v10.3.3` tag that has been created in core master. At that tag is the needed Behat-Mink acceptance test dependency changes that are needed. There is only test changes 
There is no actual release or QA tarball built off that, because there is probably nothing that actually needs that.
The tag is useful for repos like this to fetch the appropriate working version of test tools.